README.md

```diff
@@ -110,7 +110,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? "moon goodnight"
+- What is mystery6("Ada Developers Academy")? "Developers Academy Ada"
+- What is mystery6("Hi, there!")? "there! Hi,"
```

Comment:
Minor, but notable: there should be a leading space on each of the answers " moon goodnight", " Academy Developers Ada", " there! Hi,"